circleci.sh

```diff
@@ -1,11 +1,32 @@
 #!/bin/bash
-RUBY_VERSIONS=(2.0.0-p645 2.1.6 2.2.2)
+RUBY_VERSIONS=(2.0.0-p648 2.1.8 2.2.4)
 CIRCLE_NODE_RUBY_VERSION="${RUBY_VERSIONS[$CIRCLE_NODE_INDEX]}"
+NUM_RUBIES=${#RUBY_VERSIONS[@]}
+METRICS_BUILD_RUBY_VERSION="2.2.4"
 
-if [[ "$1" = "build" ]]
-then
+if [ $CIRCLE_NODE_TOTAL -le $NUM_RUBIES ]; then
+  echo "Cannot test $NUM_RUBIES ruby versions and metrics on $CIRCLE_NODE_TOTAL containers"
+  exit 1
+fi
+
+# Exit early if there are more containers than ruby versions to test
+if [ $CIRCLE_NODE_INDEX -gt $(($NUM_RUBIES)) ]; then
```

Comment:
What's the reason for wrapping this in `$((...))`? You didn't do it on line 7, so I wondered if maybe it was left-over code.